Jenkinsfile

```diff
@@ -47,118 +47,144 @@ properties(
-                            [$class: 'StringParameterValue', name: 'CLAMAV_JOB_NUMBER', value: "${BUILD_NUMBER}"],
-                            [$class: 'StringParameterValue', name: 'BUILD_JOB_NAME', value: "test-pipelines/${params.BUILD_PIPELINE}"],
-                            [$class: 'StringParameterValue', name: 'BUILD_JOB_NUMBER', value: "${buildResult.number}"],
-                            [$class: 'StringParameterValue', name: 'TESTS_BRANCH', value: "${params.TESTS_BRANCH}"],
-                            [$class: 'StringParameterValue', name: 'FRAMEWORK_BRANCH', value: "${params.FRAMEWORK_BRANCH}"],
-                            [$class: 'StringParameterValue', name: 'VERSION', value: "${params.VERSION}"],
-                            [$class: 'StringParameterValue', name: 'SHARED_LIB_BRANCH', value: "${params.SHARED_LIB_BRANCH}"]
-                        ]
-                    )
-                    echo "test-pipelines/${params.REGULAR_PIPELINE} #${regularResult.number} succeeded."
+                checkout scm
+
+                dir(path: 'clamav_documentation') {
+                    git(url: 'https://github.com/Cisco-Talos/clamav-documentation.git', branch: "gh-pages")
                 }
-            } catch (exc) {
-                echo "test-pipelines/${params.REGULAR_PIPELINE} failed."
-                exception = exc
-            }
-            stage("Custom Pipeline") {
-                final customResult = build(job: "test-pipelines/${params.CUSTOM_PIPELINE}",
-                    propagate: true,
-                    wait: true,
-                    parameters: [
-                        [$class: 'StringParameterValue', name: 'CLAMAV_JOB_NAME', value: "${JOB_NAME}"],
-                        [$class: 'StringParameterValue', name: 'CLAMAV_JOB_NUMBER', value: "${BUILD_NUMBER}"],
-                        [$class: 'StringParameterValue', name: 'TESTS_BRANCH', value: "${params.TESTS_CUSTOM_BRANCH}"],
-                        [$class: 'StringParameterValue', name: 'FRAMEWORK_BRANCH', value: "${params.FRAMEWORK_BRANCH}"],
-                        [$class: 'StringParameterValue', name: 'VERSION', value: "${params.VERSION}"],
-                        [$class: 'StringParameterValue', name: 'SHARED_LIB_BRANCH', value: "${params.SHARED_LIB_BRANCH}"]
-                    ]
-                )
-                echo "test-pipelines/${params.CUSTOM_PIPELINE} #${customResult.number} succeeded."
-            }
-            if(exception != null) {
-                echo "Custom Pipeline passed, but Regular pipeline failed!"
-                throw exception
+
+                dir(path: 'docs/html') {
+                    sh """# Move the clamav-documentation here.
+                    cp -r ../../clamav_documentation/* .
+                    # Clean-up
+                    rm -rf ../../clamav_documentation
+                    rm -rf .git .nojekyll CNAME Placeholder || true
+                    """
```

Comment:
This commit loses some indentation here and in other script blocks that makes it easier to see the script content separate from the jenkins command. I would prefer to keep that. 

E.g.
```jenkinsfile
                    sh """# Move the clamav-documentation here.
                        cp -r ../../clamav_documentation/* .
                        # Clean-up
                        rm -rf ../../clamav_documentation
                        rm -rf .git .nojekyll CNAME Placeholder || true
                        """
```
or maybe

```jenkinsfile
                    sh """
                        # Move the clamav-documentation here.
                        cp -r ../../clamav_documentation/* .
                        # Clean-up
                        rm -rf ../../clamav_documentation
                        rm -rf .git .nojekyll CNAME Placeholder || true
                    """
```